samples/operate-first/fybrik-vault.yaml

```diff
@@ -0,0 +1,29 @@
+# Source: vault/templates/vault-cluster-role.yaml
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vault-secrets-manager
```

Comment:
@eletonia this  ClusterRole should not be generated, does fybrik-vault.yaml was updated after the rebase? (I do not see it is generated when I try your branch)

Reply:
Just regenerated it, this cluster role is no longer there